docs/SONOMA-DROP.md

```diff
@@ -0,0 +1,117 @@
+
+## Versioning
+
+With v1.0.0, we'll be switching to a proper major, minor and bug fix system ([Semantic Versioning](https://semver.org/)). This means the coming release will be version 1.0.0, and future releases plan to follow this scheme:
+
+- First digit: Major changes, including new OS support, API changes, and significant patch set changes, etc
+- Second digit: Minor changes, including incoming OS update fixes, minor patch set changes, etc
+- Third digit: Bug fixes, primarily hot fixes either due to a regression in prior release or resolving issues in already released OS updates
+
+
+## Newly dropped hardware
+
+* MacBook10,1:       MacBook (Retina, 12-inch, 2017)
+* MacBookPro14,1:    MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports)
+* MacBookPro14,2:    MacBook Pro (13-inch, 2017, Four Thunderbolt 3 Ports) 
+* MacBookPro14,3:    MacBook Pro (15-inch, 2017)
+* iMac18,1:          iMac (21.5-inch, 2017)
+* iMac18,2:          iMac (Retina 4K, 21.5-inch, 2017)
+* iMac18,3:          iMac (Retina 5K, 27-inch, 2017)
+
+## Current status
+
+OpenCore Legacy Patcher v1.0.0 will support Sonoma for all models normally supported by the Patcher, however some challenges remain. You can find information about them below.
+
+## Issues
+
+* [Widgets freezing on 3802 systems](#widgets-freezing-on-3802-systems)
+* [T1 Security chip](t1-security-chip)
+* [USB 1.1 (OHCI/UHCI) Support](#usb-11-ohciuhci-support)
+
+### Widgets freezing on 3802 systems
+
+Opening the widget board on 3802 based systems can freeze the system.
+
+::: details Graphics cards belonging under 3802 include: (click to expand)
+
+__Intel Ivy Bridge__
+
+```sh
+Applicable Models:
+- MacBookAir5,x
+- MacBookPro9,x
+- MacBookPro10,x
+- iMac13,x
+- Macmini6,x
+```
+__Intel Haswell__
+
+```sh
+Applicable Models:
+- MacBookAir6,x
+- MacBookPro11,x
+- iMac14,x
+- iMac15,1 (internal, headless iGPU)
+- Macmini7,1
+```
+
+__NVIDIA Kepler__
+```sh
+Applicable Models:
+- MacBookPro9,1
+- MacBookPro10,1
+- MacBookPro11,3
+- iMac13,x
+- iMac14,x
+```
+::: 
+
+### T1 Security chip
+
+Sonoma has removed support for T1 chips found in most 2016 and 2017 Macs. Therefore on these systems, the following will not function:
+
+* Enable or disable FileVault
+* Open the Password Settings window
+* Add fingerprints (if upgrading, existing fingerprints will be deleted)
+* Add cards to Apple Pay
+
+::: warning ATTENTION: Additionally, due to Sonoma no longer caring about T1, erasing the entire drive will remove the T1 firmware and it won't be reinstalled.
+:::
+No known solution to resolve supporting, a significant amount of time will be required to understand how both the T1 stack works, as well as where the core issue lies for support.
+
+[More information here](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1103)
+
+### USB 1.1 (OHCI/UHCI) Support
+
+For Penryn systems and pre-2013 Mac Pros, USB 1.1 support was outright removed in macOS Ventura and naturally this continues in Sonoma. 
```

Comment:
2009 Xserve as well, I suggest "Penryn-based and Nehalem/Westmere-based systems"